prow/crier/reporters/gcs/reporter.go

```diff
@@ -197,7 +197,7 @@ func (gr *gcsReporter) reportFinishedJob(ctx context.Context, log *logrus.Entry,
 
 func (gr *gcsReporter) reportProwjob(ctx context.Context, log *logrus.Entry, pj *prowv1.ProwJob) error {
 	// Unconditionally dump the prowjob to GCS, on all job updates.
```

Comment:
Style nit: Use consistent language. E.g. "prowjob" vs "prow job" (I think "Prow Job" is better but I forget what is preferred in the codebase), "GCS" vs "gcs". Also JSON should always be capitalized (such as in your commit message).

Reply:
Will do.

Reply:
I've fixed the commit messages. It would be great to have a style guide to refer for such comments.

There is a lot of use of "prowjob" as a noun in the code surrounding my changes, which is likely why I've written as I have. And for the record, I didn't write the lines you noticed.

I believe we should use "ProwJob" when referring to the struct (as here), and "prow job" (or simply "job") when referring to the concept of a job run by Prow.